app/views/comments/_form.html.erb

```diff
@@ -28,73 +27,90 @@
+        <div class="comment-form__toolbar">
+          <div class="editor-image-upload">
+            <input type="file" id="image-upload-main" name="file" accept="image/*" style="display:none">
+            <button type="button" class="crayons-btn crayons-btn--s crayons-btn--icon-left crayons-btn--ghost-dimmed" onclick="handleImageUpload(event,'main')">
+              <%= inline_svg_tag("image.svg", aria: true, class: "crayons-icon", title: "Image") %>
+              <span class="hidden s:inline-block">Upload image</span>
+            </button>
+            <label class="image-upload-file-label" id="image-upload-file-label-main"></label>
+            <input type="submit" id='image-upload-submit-main' value="Upload" style="display:none">
+            <input class="hidden" id="uploaded-image-main" />
+          </div>
+
+          <button type="button" class="crayons-btn crayons-btn--s crayons-btn--icon-left crayons-btn--ghost-dimmed response-templates-button" title="Use a response template" data-has-listener="false" data-form-id="new_comment">
+            <%= inline_svg_tag("templates.svg", aria: true, class: "crayons-icon", title: "Templates") %>
+            <span class="hidden s:inline-block">Templates</span>
+          </button>
+
+          <a href="/p/editor_guide" class="crayons-btn crayons-btn--ghost-dimmed crayons-btn--icon crayons-btn--s ml-auto" target="_blank" rel="noopener" title="Markdown Guide">
+            <%= inline_svg_tag("info.svg", aria: true, class: "crayons-icon", title: "Editor guide") %>
+          </a>
+        </div>
+      </div>
+
+      <div class="response-templates-container crayons-card crayons-card--secondary p-4 mb-4 comment-form__templates hidden">
         <header>
-          <button type="button" class="personal-template-button active" data-target-type="personal" data-form-id="new_comment">PERSONAL</button>
-          <button type="button" class="moderator-template-button hidden" data-target-type="moderator" data-form-id="new_comment">MODERATOR</button>
+          <button type="button" class="personal-template-button active" data-target-type="personal" data-form-id="new_comment">Personal</button>
+          <button type="button" class="moderator-template-button hidden" data-target-type="moderator" data-form-id="new_comment">Moderator</button>
         </header>
+
         <img class="loading-img hidden" src="<%= asset_path("loading-ellipsis.svg") %>" alt="loading">
+
         <div class="personal-responses-container">
           <%# filled by JS %>
         </div>
         <div class="moderator-responses-container hidden">
           <%# filled by JS %>
         </div>
-        <a target="_blank" rel="noopener nofollow" href="<%= user_settings_path(tab: "response-templates") %>" class="mod-response-create-new">
+
+        <a target="_blank" rel="noopener nofollow" href="<%= user_settings_path(tab: "response-templates") %>">
           Create new template
```

Comment:
I think that instead of saying "Create new template", we can just say "Create template". There is no need for the adjective "new" here.

Reply:
Very good suggestion! I like short labels :)

Reply:
Me too!